Engine/BaseAssets/Components/Postprocessing/Bloom/PostProcessEffect_Bloom.cs

```diff
@@ -3,13 +3,25 @@
 using SharpDX.DXGI;
 using SharpDX.Direct3D11;
 using System;
-using System.Windows.Documents;
 using System.Collections.Generic;
 
-namespace Engine.BaseAssets.Components.Postprocessing.Bloom
+namespace Engine.BaseAssets.Components.Postprocessing
 {
-    public class PixelShaderBloom : PostProcessEffect
+    public class PostProcessEffect_Bloom : PostProcessEffect
     {
+        public int Iterations 
+        {
+            get => iterations;
+            set
+            {
+                if(value < 1 || value > MaxIterationsCount)
+                    throw new ArgumentOutOfRangeException("Iterations", "Iterations must be greater than 1 and less or equal than MaxIterationsCount");
```

Comment:
MaxIterationsCount is referred, but it's private, make it public static const instead

Reply:
Done